SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -142,9 +142,16 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 
 	if (LoadedWorld->URL.HasOption(TEXT("locator")))
```

Comment:
I think this should maybe be changed to `legacylocator` and `locator` respectively.

Reply:
I agree, however, this will break anyone with existing code that uses the locator in a URL. Not sure anyone is relying on it since I suspect NWX are the only ones actually doing that, but it's worth taking this into account. It'll break NWX portals temporarily.

Reply:
I didn't want to change the `"locator"` param here since there was the potential for people to be using it already.  If nobody is using it, I can change it.

We could also look at checking for `deployment=` for the legacy flow, and `playeridentity=` for the new flow instead of needing an extra option that is otherwise unused.

Reply:
I believe no one is using it to the extent that it's critically necessary. NWX are probably the biggest users of this flow at the moment.

Reply:
I like the idea of making it explicit which flow to follow, but I'll defer to the GDK team.

If nobody else is using the `locator` parameter then it's fine, I just wanted to be cautious.

Reply:
I've made these changes now.  Is there any documentation I'll need to follow up and update when this is merged in?

Reply:
Yep, take a look at `SpatialGDK/Documentation/content/map-travel.md`

Reply:
I've updated the documentation in this commit: https://github.com/spatialos/UnrealGDK/pull/615/commits/e44554a5df167f5b9c0b0a2b98c62a7399a259c6